domains/bryce.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "website to describe me",
+    "repo": "https://github.com/github-username/github-repository",
+    "owner": {
+        "username": "GitHubFanTest",
+        "email": "feelingbloxyburger24@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://github.com/GitHubFanTest/bryce-website"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.